website/blog/authors.yml

```diff
@@ -27,9 +34,73 @@ chi:
   image_url: https://github.com/chilagrow.png
   page: true
 
+dmitry:
+  name: Dmitry Eremenko
+  page: true
+
 elena:
   name: Elena Grahovac
   title: FerretDB Team
   url: https://www.linkedin.com/in/grahovac/
   image_url: https://github.com/rumyantseva.png
   page: true
+
+farkasp:
```

Comment:
Let's not change existing aliases. Could you please keep `peter` like before. Please make sure the corresponded blog posts are correctly assigned.

Reply:
Let's keep this as it was using`peter` and change Peter Benei to `peterbenei`.

Reply:
Got it